nrf_wifi/fw_if/umac_if/src/tx.c

```diff
@@ -1830,8 +1830,7 @@ enum nrf_wifi_status nrf_wifi_fmac_start_rawpkt_xmit(void *dev_ctx,
 		 * return failure as network buffer and device
 		 * context are NULL
 		 */
-		def_dev_ctx->raw_pkt_stats.raw_pkt_send_failure += 1;
-		return NRF_WIFI_STATUS_FAIL;
+		goto fail;
```

Comment:
Why do we need a `fail` condition? We could just remove the `def_dev_ctx->raw_pkt_stats.raw_pkt_send_failure += 1;` and return `NRF_WIFI_STATUS_FAIL` from here.

Reply:
I wished to have a FAIL condition as there was an earlier comment in my previous review that we should be returning at the end of the function. hence the goto. I see tx_cmd_prepare, tx_process functions doing the same. Now, if the comment is the opposite of what we are supposed to follow. 

Do let me know what you think and what we follow. I will make the changes accordingly.

Thanks and regards
Vivek